src/features/defi/providers/thorchain-savers/components/ThorchainSaversManager/Overview/ThorchainSaversOverview.tsx

```diff
@@ -85,6 +90,22 @@ export const ThorchainSaversOverview: React.FC<OverviewProps> = ({
   const assets = useAppSelector(selectAssets)
   const asset = useAppSelector(state => selectAssetById(state, assetId))
 
+  useEffect(() => {
+    ;(async () => {
+      if (!(asset && assetId)) return
+
+      const maybeQuote = await getMaybeThorchainSaversDepositQuote({
+        asset,
+        amountCryptoBaseUnit: THORCHAIN_SAVERS_DUST_THRESHOLDS[assetId],
+      })
+      if (
+        maybeQuote.isErr() &&
+        maybeQuote.unwrapErr().includes('add liquidity rune is more than bond')
```

Comment:
do they have a custom error code out of curiosity for this hard cap case? only concern is I feel like error text has higher chance to change than an error code

Reply:
Unfortunately not, if we hit `/thorchain/quote/saver/deposit?asset=BTC.BTC&amount=10000000`, we get a GraphQL style 200, no HTTP error code, no `code` property, only a message
<img width="1178" alt="image" src="https://github.com/shapeshift/web/assets/17035424/f79d09cb-7ac9-4076-89ad-1957b2d09d74">